2022-03-06 12:57:53,260 INFO    MainThread:697181 [wandb_setup.py:_flush():71] setting env: {}
2022-03-06 12:57:53,260 INFO    MainThread:697181 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-06 12:57:53,260 INFO    MainThread:697181 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-06 12:57:53,260 INFO    MainThread:697181 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220306_125753-b0yb9t3r/logs/debug.log
2022-03-06 12:57:53,260 INFO    MainThread:697181 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220306_125753-b0yb9t3r/logs/debug-internal.log
2022-03-06 12:57:53,260 INFO    MainThread:697181 [wandb_init.py:init():404] calling init triggers
2022-03-06 12:57:53,260 INFO    MainThread:697181 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-03-06 12:57:53,260 INFO    MainThread:697181 [wandb_init.py:init():460] starting backend
2022-03-06 12:57:53,260 INFO    MainThread:697181 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-06 12:57:53,267 INFO    MainThread:697181 [backend.py:ensure_launched():216] starting backend process...
2022-03-06 12:57:53,270 INFO    MainThread:697181 [backend.py:ensure_launched():222] started backend process with pid: 697190
2022-03-06 12:57:53,271 INFO    MainThread:697181 [wandb_init.py:init():469] backend started and connected
2022-03-06 12:57:53,272 INFO    MainThread:697181 [wandb_init.py:init():533] updated telemetry
2022-03-06 12:57:53,284 INFO    MainThread:697181 [wandb_init.py:init():563] communicating current version
2022-03-06 12:57:54,291 INFO    MainThread:697181 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-06 12:57:54,291 INFO    MainThread:697181 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-06 12:57:54,587 INFO    MainThread:697181 [wandb_init.py:init():606] starting run threads in backend
2022-03-06 12:57:57,240 INFO    MainThread:697181 [wandb_run.py:_console_start():1810] atexit reg
2022-03-06 12:57:57,240 INFO    MainThread:697181 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-06 12:57:57,240 INFO    MainThread:697181 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-06 12:57:57,241 INFO    MainThread:697181 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-06 12:57:57,242 INFO    MainThread:697181 [wandb_init.py:init():633] run started, returning control to user process
2022-03-06 12:57:59,396 INFO    MainThread:697181 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-06 12:57:59,397 INFO    MainThread:697181 [wandb_run.py:_restore():1752] restore
2022-03-06 12:58:02,254 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-06 12:58:02,587 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-06 12:58:02,689 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15576
}

2022-03-06 12:58:02,792 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15576
}

2022-03-06 12:58:02,894 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15576
}

2022-03-06 12:58:02,996 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15576
  total_bytes: 15576
}

2022-03-06 12:58:03,098 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15576
  total_bytes: 15576
}

2022-03-06 12:58:03,201 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15576
  total_bytes: 15576
}

2022-03-06 12:58:03,303 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15576
  total_bytes: 15576
}

2022-03-06 12:58:03,406 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15576
  total_bytes: 15576
}

2022-03-06 12:58:03,508 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15576
  total_bytes: 15576
}

2022-03-06 12:58:03,985 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15576
  total_bytes: 15576
}

2022-03-06 12:58:04,601 INFO    MainThread:697181 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15576
  total_bytes: 15576
}
local_info {
}

2022-03-06 12:58:05,664 INFO    MainThread:697181 [wandb_run.py:_append_files():2180] logging synced files
